### PR TITLE
Jamie find cargo

### PR DESCRIPTION
**Pull request by lchangGH**
Modify code to identify cargo, calculate distance and angle

### CODE REVIEW

File: MergeFRCPipeline.py [hunk 1]
```diff
@@ -380,7 +382,9 @@ def switchCam():
     #PipeLine Table Values, Unique for Each PipeLine
     networkTableVisionPipeline.putBoolean("Driver", False)
     networkTableVisionPipeline.putBoolean("Tape", TapeEnabled)
-    networkTableVisionPipeline.putBoolean("PowerCell", PowerCellEnabled)
+    networkTableVisionPipeline.putBoolean("Cargo", CargoEnabled)
+    networkTableVisionPipeline.putBoolean("Red", RedEnabled)
+    networkTableVisionPipeline.putBoolean("RedBlue", BlueEnabled)
```
Comment (jogray72) [inline MergeFRCPipeline.py]: Should this be "Blue" instead of "RedBlue"?

File: VisionConstants.py [hunk 2]
```diff
@@ -20,21 +20,21 @@
 horizontalView = math.atan(math.tan(diagonalView / 2) * (horizontalAspect / diagonalAspect)) * 2
 verticalView = math.atan(math.tan(diagonalView / 2) * (verticalAspect / diagonalAspect)) * 2
 
-# MAY CHANGE IN FUTURE YEARS! This is the aspect ratio used in 2020
-image_width = 640 # 4
-image_height = 480 # 3
+# MAY CHANGE IN FUTURE YEARS! This is the aspect ratio used in 2022
+image_width = 720 # 4
```
Comment (jogray72) [inline VisionConstants.py]: Isn't the image Width 1280, and the height 720?    (1280 X 720).

**Review by jogray72 — COMMENTED** [review]
There are a few comments you need to look at (Resolution width and height looks reversed, and network table "redblue" doesn't seem correct.

**Comment by jogray72** [comment]
The other issue is that this causes a Merge conflict.

File: MergeFRCPipeline.py (hunk 1 above)
Comment (lchangGH) [reply MergeFRCPipeline.py]: You are correct

**Review by jogray72 — COMMENTED** [review]
Looks good.   Just resolve the Merge conflict now by updating the pipeline config changes.

**Pull request closed without merging**